pkg/glance/volumes.go

```diff
@@ -70,11 +71,13 @@ func GetVolumes(name string, pvcName string, extraVol []glancev1.GlanceExtraVolM
 			vm = append(vm, vol.Volumes...)
 		}
 	}
+	secretConfig, _ := GetConfigSecretVolumes(secretNames)
+	vm = append(vm, secretConfig...)
```

Comment:
Minor comment: Should we extract this in separate small method which will return vm and call it here and at line 104 instead duplicating the code?

Reply:
so I think this is not helping w/ de-duplication. In order to de-duplicate some code I created that function that returns you both `corev1.Volume` and `corev1.VolumeMount`: according to the place where it's called, I can choose the object I need and "append" to the existing slice.
Saving those two lines, imo, doesn't add value, and actually hides something that we might want to see to improve readability.
If can help, in the long term, where we want to generate the config secrets in the operator, that part might go away, so I would take the code as it is (also because I has been tested) and refactor in a follow up patch that takes care about config generation and init script removal.
@konan-abhi wdyt?

Reply:
Sounds good 👍🏻